appveyor.yml

```diff
@@ -61,6 +52,9 @@ before_build:
 - createuser -U postgres test
 - psql -U postgres -c "alter user test with password 'test'" postgres
 - psql -U postgres -c "alter user test with replication" postgres
+- psql -U postgres -c "CREATE LANGUAGE plpgsql" test
+- psql -U postgres -c "CREATE EXTENSION hstore" test
+- psql -U postgres -c "CREATE EXTENSION sslinfo" postgres
```

Comment:
Why do we have to change anything for this when we're only updating the JDK version?

Reply:
Oh wait, is this because appveyor was completely broken before and now it might actually work?